Docs/Migration-INI.md

```diff
@@ -0,0 +1,854 @@
+# Migrating from older versions - INI configuration
+This guide uses [YR mod base](https://github.com/Starkku/cncnet-client-mod-base) configuration as an example. The majority of changes also applies to to non-YR client configurations.
+
+It is **highly recommended** to make a complete backup of your mod before starting.
+
+## Edit `ClientDefinitions.ini`
+1. Add `[Settings]->UnixLauncherExe=Launcher.sh` (script file name can be anything)
+2. Create `Launcher.sh` in root directory:
+```sh
+#!/bin/sh
+
+cd "$(dirname "$0")"
+dotnet Resources/Binaries/UniversalGL/clientogl.dll "$@"

```

Comment:
We have migrated the client to .NET 4.8/8.0. The folder structure is updated. .NET8 builds reside in `BinariesNET8` folder.

Reply:
I have been told that .NET 8.0 version won't be shipped to users.

Reply:
I will give a precise description: net 8 builds **are** shipped to users, unless modders **actively** remove the `BinariesNET8` folder. Of course we can discuss and adjust the default behavior but this is the current behavior